src/plugins/custom_import_map/public/components/empty_prompt.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Fragment } from 'react';
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+export interface EmptyPromptProps {
+  iconType: string;
+  title: string;
+  bodyFragment: string;
+  actions: any;
+}
+
+const EmptyPrompt = (props: EmptyPromptProps) => {
+  return (
+    <EuiEmptyPrompt
+      iconType={props.iconType}
+      title={<h2>{props.title}</h2>}
+      body={
+        <Fragment>
+          <p>{props.bodyFragment}</p>
+        </Fragment>
+      }
+      actions={props.actions}
+    />
+  );
+};
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
Nit: instead of disabling this, just export the `EmptyPrompt` component

e.g.:

```
export const EmptyPrompt = (props: EmptyPromptProps) => {
```